tools/tcprtt_example.txt

```diff
@@ -40,16 +40,65 @@ also shows unstable TCP RTT. So in this situation, we need to make sure the
 quality of network is good or not firstly.
 
 
-Use filter for address and(or) port. Ex, only collect source address 192.168.122.200
-and destination address 192.168.122.100 and destination port 80.
+Use filter for address and(or) port. Ex, only collect local address 192.168.122.200
+and remote address 192.168.122.100 and remote port 80.
 # ./tcprtt -i 1 -d 10 -m -a 192.168.122.200 -A 192.168.122.100 -P 80
 
 
+Tracing at server side, show each clients with its own histogram.
+For example, run tcprtt on a storage node to show initiators' rtt histogram:
+# ./tcprtt -i 1 -m --lport 3260 --byraddr
+Tracing TCP RTT... Hit Ctrl-C to end.
+
+srtt = Remote Address: 10.131.90.18
```

Comment:
Could you change the head to 
  Remote Address = 10.131.90.18
This is to conform to other bcc tool histogram output (e.g., biolatency.py).

Reply:
Fixed as you suggested.